src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/BlahVetCreation.java

```diff
@@ -0,0 +1,249 @@
+                return variant.getFilters();
+            }
+        },
+
+        CALL("call") {
+            public String getColumnValue(final VariantContext variant) throws IOException {
+                return variant.getGenotype(0).getGenotypeString(); // TODO double check
+            }
+        },
+
+        CALL_NAME("call.name") {
+            public String getColumnValue(final VariantContext variant) throws IOException {
+                return variant.getGenotype(0).getGenotypeString(); // TODO double check
+            }
+        },*/
+
+        CALL_GENOTYPE("call.genotype") {
+            public String getColumnValue(final VariantContext variant) throws IOException {
+                return variant.getGenotype(0).getGenotypeString(); // TODO double check
+            }
+        },
+
+         /*CALL_PHASESET("call.phaseset") {
+            public int[] getColumnValue(final VariantContext variant) throws IOException {
+                return variant.getGenotype(0).get .....// TODO lookup
+            }
+        },
+
+        CALL_AD("call.AD") {
+            public Object getColumnValue(final VariantContext variant) throws IOException {
+                return variant.getGenotype(0).getAD();
+            }
+        },
+
+        CALL_DP("call.DP") {
+            public Object getColumnValue(final VariantContext variant) throws IOException {
+                return variant.getGenotype(0).getDP();
+            }
+        },
+
+        CALL_GQ("call.GQ") { // Required
+            if (variant.getGenotype(0).getGQ().equals(null)) {
+               throw new IllegalArgumentException("Cannot be missing required value for call.GQ");
+            }
+            public Object getColumnValue(final VariantContext variant) throws IOException {
+                return variant.getGenotype(0).getGQ();
+            }
+        },
+
+        CALL_PGT("call.PGT") {
+            public Object getColumnValue(final VariantContext variant) throws IOException {
+                return variant.getGenotype(0).getAnyAttribute("PGT");// TODO lookup -- should be in Format -- cast to int
+            }
+        },
+
+        CALL_PID("call.PID") {
+            public Object getColumnValue(final VariantContext variant) throws IOException {
+                return variant.getGenotype(0).getAnyAttribute("PID");// TODO lookup -- should be in Format
+            }
+        },*/
+
+
+        CALL_PL("call.PL") {
+            public String getColumnValue(final VariantContext variant) throws IOException {
+                return variant.getGenotype(0).getPL().toString();
+            }
+        /*},
+
+        AS_SB_TABLE("AS_SB_TABLE") {
+            public String getColumnValue(final VariantContext variant) throws IOException {
+                return variant.get .....// TODO lookup
+            }*/
+        };
+
+        private String value;
+
+        HeaderFieldEnum(String value) {
+            this.value = value;
```

Comment:
we should think about what we want the value for each enum to be.  Maybe it should be the column name in the schema?